docs/general/data-structures.md

```diff
@@ -7,6 +7,8 @@ The provides labels can be of the type string or number to be rendered correctly
 
 ## Primitive[]
 
+Namespace: `data.datasets[]`
+
 ```javascript
 data: [20, 10],
```

Comment:
Might be a good idea to change this one also to a small chart config as you have done for yhe Object data type. Since now it almost looks like the data and labels are on the same level